bgpd/bgp_evpn_vty.c

```diff
@@ -4344,9 +4344,9 @@ DEFUN(show_bgp_l2vpn_evpn_vni,
 								 : "Disabled");
 			json_object_string_add(
 				json, "flooding",
-				bgp_evpn->vxlan_flood_ctrl
-						== VXLAN_FLOOD_HEAD_END_REPL
-					? "Head-end replication"
+				bgp_evpn->vxlan_flood_ctrl ==
```

Comment:
This will break backwards compatability.  I need a good argument why this is ok to break people's screen scrapers/json handling scripts.  If we can't have that we need an upgrade path for the operator.  @chiragshah6 how likely is someone actually using this?

Reply:
This is one of the primary command to look for evpn. 
This particular change keeps consistency with respect to other evpn feature knobs status, 
```
  "advertiseGatewayMacip":"Disabled",
  "advertiseSviMacIp":"Disabled",
  "advertiseAllVnis":"Enabled",
  "flooding":"Enabled",    <----- aligns to other feature knobs
  ```
But for backward compatibility, let's introduce a new field.
For screen scrapping:
```
  BUM flooding: Head-end replication
  flooding: Enabled <<< new field
 ```  
json:
```
"vxlanFlooding: "Enabled",  <<<<< new field
"flooding": "Head-end replication",
```

Reply:
This is one of the primary command to look for evpn.
This particular change keeps consistency with respect to other evpn feature knobs status,

  "advertiseGatewayMacip":"Disabled",
  "advertiseSviMacIp":"Disabled",
  "advertiseAllVnis":"Enabled",
  "flooding":"Enabled",    <----- aligns to other feature knobs

But for backward compatibility, let's introduce a new field.
For screen scrapping:

  BUM flooding: Head-end replication
  flooding: Enabled <<< new field

json:

"vxlanFlooding: "Enabled",  <<<<< new field
"flooding": "Head-end replication",

Reply:
Adding a new field to VTY and JSON.